README.md

```diff
@@ -193,6 +194,8 @@ namespace BlingUtilsTest
     {
         static void Main(string[] args)
         {
+            Console.OutputEncoding = Encoding.Unicode;

```

Comment:
System.Text.Encoding.UTF8 should be better, because the sample works with UTF8 (`byte[] inBytes = System.Text.Encoding.UTF8.GetBytes(input);`)

Reply:
Good point. I thought Unicode worked better with console or redirected output, but after re-testing it seems UTF8 is better, so switched to that.